nativeauthenticator/orm.py

```diff
@@ -13,6 +16,12 @@ class UserInfo(Base):
     password = Column(LargeBinary, nullable=False)
     is_authorized = Column(Boolean, default=False)
     email = Column(String)
+    otp_secret = Column(String(10))
```

Comment:
What does the '10' refer to here?

Reply:
the token is created with size 10, so I limited here